README.md

```diff
@@ -79,11 +69,18 @@ or on windows:
 gradlew jpackage
 ```
 
+Note that *jpackage* creates the installer for the platform that builds it.
+Building cross-platform installers is not supported by Java's jpackage facility.
+You may need to have developer tools for the platform in order to build the installer.
+See Java's [jpackage documentation](https://docs.oracle.com/en/java/javase/14/jpackage/packaging-overview.html) for more details of tool requirements.
+
 ## Testing logisim-evolution
 
-As logisim-evolution needs updates (new features and patches) and currently lacks unit tests, the *development* branch was created.
+As logisim-evolution is often updated, the development branch, *develop*, was created.
 The goal of this branch is to add new features/patches without affecting the release on branch master.
-Users who are willing test new features should checkout the development branch. The feedback from users is really appreciated as it makes logisim-evolution better. Feel free to use the issue tab to report bugs and suggest features.
+Users who are willing test new features should checkout the develop branch.
```

Comment:
... willing _to_ test...

Reply:
Thank you, @maehne. Fixed.

Reply:
Thank you for reacting so quickly to my feedback! I tested your changes and they work on my side. One minor issue I noted, but don't know how difficult it is to fix, is that the app bundle is now called `logisim-evolution.app` (lower case), but inside the application `Logisim-evolution` (upper case) is consistently used. I had not yet time to check whether the new Window MSI installer and DEP/RPM packages for Linux are affected by a similar issue. For the command line, the executable should stay `logisim-evolution` though.

@BFH-ktt1: Independent of this minor issue, I think this PR is in good shape to be merged.

Reply:
I can change the name just for jpackage to Logisim-evolution and get that to work (without changing the jar name). I could change it just for the Mac by moving that parameter setting into the platform-specific portion, or change if for all platforms. I don't have the other platforms so I can test those. I like the idea of changing it. Which way should I try?

Reply:
I have a working version on the Mac with the name Logisim-evolution. I moved the --name parameter to the platform-specific portion so the other platforms should not be changed. Let me know if I should add that commit here or factor it back out so you can test it on the other platforms.

Reply:
@davidhutchens , when I have some spare minutes in the coming days I will test it for the other two platforms (as I have all three of them). For the moment I will pull this request to have the base for the modifications.